src/index.html

```diff
@@ -17,11 +17,388 @@
+          />
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                href="#Apple"
+                class="nav__link is-active"
+              >
+                Apple
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                href="#Samsung"
+                class="nav__link"
+              >
+                Samsung
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                href="#Smartphones"
+                class="nav__link"
+              >
+                Smartphones
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                href="#Laptops & Computers"
+                class="nav__link"
+                data-qa="nav-hover"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                href="#Gadgets"
+                class="nav__link"
+              >
+                Gadgets
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                href="#Tablets"
+                class="nav__link"
+              >
+                Tablets
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                href="#Photo"
+                class="nav__link"
+              >
+                Photo
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                href="#Video"
+                class="nav__link"
+              >
+                Video
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </header>
+      <main class="content">
+        <div class="catalog">
+          <div
+            class="card"
+            data-qa="card"
+          >
+            <img
+              class="card__image"
+              src="./images/imac.jpeg"
+              alt="image"
+            />
+            <div class="card__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </div>
+            <div class="card__code">Product code: 195434</div>
+            <div class="card__rating">
+              <div class="stars stars--star">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <div class="card__reviews">Reviews: 5</div>
+            </div>
+            <div class="card__price">
+              <div class="card__text">Price:</div>
+              <div class="card__numbers">$2,199</div>
+            </div>
+            <div
+              class="card__buy"
+              data-qa="card-hover"
```

Comment:
There is inconsistency in the `data-qa` attribute values for the `card__buy` divs. The first card uses `data-qa="card-hover"`, while the others do not have this attribute. Ensure that all cards use the same attribute value if they are meant to serve the same purpose.